src/app/post-thread-page/post-thread/post-thread.component.ts

```diff
@@ -60,6 +60,7 @@ export class PostThreadComponent {
     return new Datasource<IAdapter<any>>({
       get: (index, count, success) => {
         const comments = this.currentPost.Comments || [];
+        this.titleService.setTitle(this.currentPost.ProfileEntryResponse.Username + " on BitClout");
```

Comment:
Does this need to be here? I see that we set the title in `ngOnInit`

Reply:
Yes it required to be here otherwise page title wont get updated, it will show the default one as "`Welcome to Bitclout`" instead of "`username at Bitclout`".
I think okay to remove this line from `ngOnInit` which is redundant in this case.

Reply:
I see - okay this should live in the function `refreshPosts` instead - This `get` is called frequently by the infinite scroller, to get the next page of data, whereas refreshPosts is only called once per post.